homeassistant/components/openweathermap/strings.json

```diff
@@ -30,5 +30,11 @@
         }
       }
     }
+  },
+  "issues": {
+    "deprecated_v25": {
+      "title": "OpenWeatherMap API V2.5 deprecated",
+      "description": "OWM API v2.5 will be closed in June 2024; migrate all your OpenWeatherMap integration to mode v3.0."
```

Comment:
We should tell them that they need to be sure to get v3 enabled on their account before switching or it will not work

Reply:
Slightly updated